### PR TITLE
[Relay][Doc] Separate arguments types formatting with comma

### PR DESCRIPTION
**Pull request by wweic**
Thanks for contributing to TVM!   Please refer to guideline https://docs.tvm.ai/contribute/ for useful information and tips. After the pull request is submitted, please request code reviews from [Reviewers](https://github.com/dmlc/tvm/blob/master/CONTRIBUTORS.md#reviewers).

### CODE REVIEW

**Comment by wweic** [comment]
@jroesch please help take a look.

File: src/relay/op/type_relations.cc [hunk 1]
```diff
@@ -90,8 +90,8 @@ bool BroadcastRel(const Array<Type>& types,
                   const Attrs& attrs,
                   const TypeReporter& reporter) {
   CHECK_EQ(types.size(), 3);
-  RELAY_LOG(INFO) << "In1: " << types[0] << "In2: " << types[1]
-                  << "Out: " << types[2] << std::endl;
+  RELAY_LOG(INFO) << "In1: " << types[0] << ",In2: " << types[1]
```
Comment (MarisaKirisame) [inline src/relay/op/type_relations.cc]: maybe ", In2:" to keep consistency with https://github.com/dmlc/tvm/blob/master/src/relay/ir/expr.cc?

Comment (wweic) [reply src/relay/op/type_relations.cc]: @MarisaKirisame done.

**Comment by tqchen** [comment]
Thanks, @wweic @MarisaKirisame , this is merged

**Pull request merged**